style.css

```diff
@@ -0,0 +1,255 @@
+@import url("https://fonts.googleapis.com/css2?family=Alumni+Sans+Collegiate+One&family=Montserrat:wght@100;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Alumni+Sans+Collegiate+One&family=Crete+Round&family=Montserrat:wght@100;500&display=swap");
+
+* {
+  margin: 0;
+
+  /* border: 1px solid black; */
+}
+
+/* class to import */
+.white {
+  color: white;
+}
+
+.blue-1 {
+  color: #091e42;
+}
+
+.blue-2 {
+  color: #091e42;
+}
+
+.yelow {
+  color: #ff6b00;
+}
+
+.bg-btn {
+  background-color: rgba(255, 255, 255, 0.24);
+}
+
+.font-montesori {
+  font-family: "Montserrat", sans-serif;
+}
+
+/* Content of the page */
+.nav-container {
+  display: flex;
+  width: auto;
+  height: auto;
+  align-items: center;
+  background-color: #3c3a39;
+  justify-content: space-between;
+  padding: 20px;
+  position: sticky;
+  z-index: 1;
+  top: 0;
+  font-family: "Montserrat", sans-serif;
+}
```

Comment:
- Kindly, when you wrap the hamburger to the nav you lose your design, please check your design and stick to the Figma design.

🤝️


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/205422648-0e473b4d-05b6-4993-8c17-b0a22e5b3abc.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/205422649-c90eab25-c7bf-4d08-a1b8-0bd08d49876e.png" width="200">|